modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,35 @@
+[id="update-service-install-web-console"]
+= Installing the {product-title} Update Service using the web console
+include::modules/common-attributes.adoc[]
+
+You can use the {product-title} web console to install the {product-title} Update Service operator.
+
+== Procedure
+
+To install the {product-title} Update Service operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose  *{product-title} Update Service* from the list of available Operators, and click *Install*.
```

Comment:
```suggestion
. Choose *{product-title} Update Service* from the list of available Operators, and click *Install*.
```
Is this actually what shows up in the UI?

Reply:
Done with adf5ca0732 -> 2c77ff9ce2

> Is this actually what shows up in the UI?

[Should be][1]?

[1]: https://catalog.redhat.com/software/operators/detail/5f0f35842991b4207fcdb202/deploy